client/add/add.module.js

```diff
@@ -20,4 +20,17 @@ ng.module('porybox.add', ['porybox.box', 'porybox.pokemon', 'ngMessages', 'ngFil
       '$scope', '$location', 'io', '$mdDialog', '$mdMedia', '$mdToast', 'errorHandler', controller
     ],
     controllerAs: 'add'
+  })
+  .component('addPokemon',
+  {
+    bindings: {
+      boxes: '=',
+      prefs: '=',
+      selected: '='
+    },
+    template: '<md-button class="md-raised md-primary" ng-click="add.pokemon($event)">Add Pokémon</md-button>',
```

Comment:
Is there a reason to have a separate component for this, rather than just embedding it in box-list.view.html?